lms/views/lti/deeplinking.py

```diff
@@ -1,5 +1,5 @@
 """
-Views for handling content item selection launches.
+Views for handling content item selection / deep linking launches.
```

Comment:
I think both terms are equivalent, maybe content-item was the first name but it looks that deeplinking is the prefered name since 1.0:

https://www.imsglobal.org/specs/lticiv1p0

```
Learning Tools Interoperability® Deep Linking
(formerly named Content-Item Message)
```

Reply:
```suggestion
Views for handling deep linking launches.
```

Maybe just go the whole way and pick a side?

Reply:
Also that link would be a killer addition to the docstring here.